apps/api/src/hyp3_api/api-spec/job_parameters.yml.j2

```diff
@@ -1,5 +1,14 @@
 components:
   schemas:
+    job_type:
+      description: Type of process to run this job.
+      type: string
+      example: RTC_GAMMA
```

Comment:
Would like to populate this using something like `job_types.values()[0]['api_job_type']`, but haven't yet found a way to do it in the limited context of a jinja template